src/konnektor/network_planners/generators/explicit_network_generator.py

```diff
@@ -25,11 +25,11 @@ def __init__(
         Parameters
         ----------
         mapper: AtomMapper
-            the atom mapper is required, to define the connection between two ligands.
+            Defines the connection between two ligands.
```

Comment:
Capitalization isn't consistent - here you capitalize, other places you dont.

Personally I prefer it if these are capitalized, that matches the `NumPy` docstyle we aim to use everywhere. I'm not sure if the non-capitalized approach folllows some other docstring style so maybe I'm missing something.

Reply:
I also prefer capitalization. I'll make a follow-up PR with just formatting to keep this one from becoming huge.